worker/dbaccessor/worker.go

```diff
@@ -15,14 +15,19 @@ import (
 	"github.com/juju/worker/v3"
 	"github.com/juju/worker/v3/catacomb"
 	"github.com/juju/worker/v3/dependency"
-	"gopkg.in/tomb.v2"
 
 	coredatabase "github.com/juju/juju/core/database"
 	"github.com/juju/juju/database/app"
 	"github.com/juju/juju/database/dqlite"
 	"github.com/juju/juju/pubsub/apiserver"
 )
 
+var (
+	// ErrTryAgain indicates that the worker should try again to start the
+	// worker.
+	ErrTryAgain = errors.ConstError("watcher trying again starting up")
```

Comment:
I'd be very clear here. Something more like: 

"DB node is nil, but worker is not dying; delaying TrackedDB start attempt"